common/queue/src/main/java/org/thingsboard/server/queue/sqs/TbAwsSqsSettings.java

```diff
@@ -42,4 +47,17 @@ public class TbAwsSqsSettings {
     @Value("${queue.aws_sqs.threads_per_topic}")
     private int threadsPerTopic;
 
+    @Value("${queue.aws_sqs.producer_thread_pool_size:0}")
+    private int threadPoolSize;
+
+    private ExecutorService producerExecutor;
+
+    @PostConstruct
+    private void init() {
+        if (threadPoolSize == 0) {
+            threadPoolSize = 50; //AmazonSQSAsyncClient.DEFAULT_THREAD_POOL_SIZE = 50;
+        }
+        producerExecutor = Executors.newFixedThreadPool(threadPoolSize, ThingsBoardThreadFactory.forName("aws-sqs-queue-executor"));
+    }
```

Comment:
Let's place the executor in `TbAwsSqsAdmin`. Having the executor in a settings class doesn't seem right.